playbook.md

```diff
@@ -69,23 +69,29 @@ If you wish to change the config in `01_defaults.env` or `03_backend.env`, you
 need to merge a change to the `services/jobrunner/defaults.env` or
 `BACKEND/backend.env`, and update the infrastructure code as above.
 
+
 ### Deploy job-runner
 
-In a clone of the [backend-server](https://github.com/opensafely-core/backend-server) repository, run:
+1. If there dependency updates, update lib
 
 ```bash
-just update-jobrunner
+just jobrunner/update-dependencies
 ```
 
-If there are dependency updates cd into `/home/opensafely/jobrunner/lib` and run:
+2. If there are migrations, run them
 
-    git pull
+```bash
+just jobrunner/migrate
```

Comment:
should this run after we update the codebase? (i.e. after `just jobrunner/deploy`) and/or can we run this automatically every time?

Reply:
We could run it automatically, its a no-op if not.

We'd probably need to test how it works with a rollback?

Reply:
It works fine, will add.

I'm slightly more cautions about the dependencies.

I wonder if we can check if they match and error out?

Reply:
ok great, let's add this & talk about dependencies in the other thread.

Reply:
I went down a bit of a dead end trying to check versions, ended up just doing the libs upgrade.

but I put it in the script, so we can handle within the confirm/exit loop.